### PR TITLE
Produce dedicated events for verification

### LINKED ISSUE #260: Devs SBAT see analytics around verification reliability
### Expected Behavior

Please describe the behavior you are expecting

### Current Behavior

What is the current behavior?

### PR DESCRIPTION
**Pull request by nambrot**
### Description

For more detailed analysis around verification performance and reliability, it would be very useful to have dedicated events for analysis.

### Tested

None

### Related issues

- Fixes #260 

### Backwards compatibility

Yes

### CODE REVIEW

File: packages/mobile/src/identity/verification.ts [hunk 1]
```diff
@@ -92,25 +91,17 @@ export function* startVerification() {
   })
 
   if (result === true) {
+    CeloAnalytics.track(CustomEventNames.verification_success)
```
Comment (jmrossy) [inline packages/mobile/src/identity/verification.ts:94]: You have not defined these in the constants.ts file above.

**Review by jmrossy — COMMENTED** [review]
I personally find this large number of diff events harder to parse but if it gives you the visibility you need than it's okay with me :)
For the future, can I recommend you run `yarn lint-checks` and `yarn test` before pushing PRs? It'll catch a lot of common mistakes

**Comment by nambrot** [comment]
Will do! I usually use VS Code's linter in the diff to find these things for me, it just has the habit on acting upon unsaved code changes which made me miss this one :)

**Review by jmrossy — COMMENTED** [review]
The verification tests will need to be updated too.
`yarn test:verbose src/identity/verification.test.ts`
You can look at how they used to be before my PR here: https://github.com/celo-org/celo-monorepo/pull/209

**Review by jmrossy — APPROVED** [review]

**Pull request merged**